### ISSUE TITLE
Streams can get mixed up or blank when using simulcast

### ISSUE THREAD

**Issue by andreisergiu98** [aws/amazon-chime-sdk-js#1805]
### What happened and what did you expect to happen?

![image](https://user-images.githubusercontent.com/11740700/142432375-0fefa845-43d9-4770-8c98-d3a1bdaa93ca.png)

Attendee 1 renders stream of attendee 2 and attendee 2 appears blank.

The issue is random, happens only locally for some attendee(s), does not revert to its initial state no matter what change (except browser refresh) any attendee makes (whether local or remote), and only happens when simulcast is active.

Upon further testing, it seems that the issue does not happen with simulcast disabled.

Mentions:
Default attendee broadcast quality: 180p at max/100Kbps
When made speaker/spotlight attendees will have their local stream stopped and then re-started in order to broadcast with a higher quality: 960p. Quality is reverted to 180p when an attendee is no longer in spotlight.
Screen-sharing is not affected and remains the only stream that is broadcast at 1080p without bandwidth restrictions
Streams are rendered using RemoteVideo from `amazon-chime-sdk-component-library-react`

What we'd like to know if this mixup can happen because of this issue https://github.com/aws/amazon-chime-sdk-js/issues/1448.

Also we tried applying the patch from the comment and while it did improve things, it seems that the mixup can still happen.

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

Join meeting, start/stop cameras or just wait.

### Amazon Chime SDK for JavaScript version

2.21.1

### What browsers are you seeing the problem on?

Chrome

### Browser version

96.0.4664.45

### Browser console logs

We were not able to reproduce the issue with logs enabled.

**Comment by hensmi-amazon** [comment]
Thanks for the report. Do you have any reproductions with logs enabled?  If not do you have a call ID with an approximate timestamp?  I don't think that issue linked would caused swapped streams, but I have been looking into logic in the transceiver controller to see if a certain order of adding and removal could cause the mapping to get out of order.

Lastly, wanted to confirm:
* If you are using the priority based policy?
* Are you subscribing to all videos at all times or just a subset (i.e. pagination).
* Are you calling `pauseReceiveVideos` on any?

**Comment by hensmi-amazon** [comment]
A possible fix is in https://github.com/aws/amazon-chime-sdk-js/pull/1831 but it is hard to be sure without aforementioned logs and answers.

**Comment by hensmi-amazon** [comment]
https://github.com/aws/amazon-chime-sdk-js/pull/1841 has also been submitted to fix the aforementioned github issue.  For testing if truly desired you could combine those two.

**Comment by hensmi-amazon** [comment]
Pardon the delay. https://github.com/aws/amazon-chime-sdk-js/pull/1831 was finally merged, will be included in 2.26 and hopefully makes this issue impossible. Please let us know if you still see it after picking up that change. I will close this issue on 2.26 release.

**Comment by hensmi-amazon** [comment]
Argh...so sorry about all of this. I had finally gotten https://github.com/aws/amazon-chime-sdk-js/pull/1831 in close to us wanting to release 2.26 and it turns out there was an additional regression that I hadn't caught (or maybe some new interaction with some change over the past 2 months), so I am just reverting in https://github.com/aws/amazon-chime-sdk-js/pull/1965 to unblock. I will get it in 2.27 though. 

Note that https://github.com/aws/amazon-chime-sdk-js/pull/1937 was included in 2.26 and fixes the more common 'blank' video with simulcast. I have not been able to reproduce the mixed up frames so its possible those are fixed too, https://github.com/aws/amazon-chime-sdk-js/pull/1831 was mainly for https://github.com/aws/amazon-chime-sdk-js/issues/1807 but it has the other benefit of making mixed up streams nearly impossible.

**Comment by hensmi-amazon** [comment]
Shipped in current release. Please open a new issue with logs if seen again.
